maze/src/common.hpp

```diff
@@ -60,16 +60,38 @@ class Coordinate
     int x;
     int y;
 
+    // Constructor
     Coordinate(int i = 0, int j = 0)
     {
        x = i; y = j;
     }
 
+    // This is a "copy constructor." It is actually identical to the default
```

Comment:
Should we include this exposition? Or just remove the unnecessary copy constructor and assignment operator?